package.json

```
@@ -127,19 +135,19 @@
     "lint:fix": "npm run lint:js:fix",
     "lint:js": "eslint --ext .js,.jsx .",
     "lint:js:fix": "eslint --fix --ext .js,.jsx .",
-    "lint:scss": "stylelint 'packages/**/src/**/*.scss' --syntax scss",
+    "lint:scss": "stylelint 'packages/**/*.scss' --syntax scss",
     "prepare-for-release": "terra prepare-for-release",
     "pretest": "npm run lint",
     "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist && npm run build && npm run build:dist",
-    "start": "tt-serve",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-prod": "tt-serve --env.disableHotReloading -p",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
+    "start": "webpack-dev-server",
+    "terra": "express-server --port $PORT --site build",
```

Sorry, I didn't give you the full snippet before.

```suggestion
    "start-heroku": "terra express-server --port $PORT --site build",
```